packages/jaeger-ui/src/components/common/vars.css

```diff
@@ -8,18 +8,25 @@ SPDX-License-Identifier: Apache-2.0
  * 
  * This file contains all color and layout variables used throughout the application.
  * New naming scheme follows semantic conventions for better maintainability.
+ * 
+ * Variables are defined on both body and .ant-layout to ensure they're available
+ * for fixed-position elements and components rendered outside the Layout.
+ * 
+ * Light mode uses fallback values since --ant-* variables may not be available
+ * on body element. Dark mode provides explicit overrides.
  */
 
+body,
 .ant-layout {
   /* ============================================
     SURFACE TOKENS (Backgrounds)
     Based on audit of background/background-color properties
     ============================================ */
 
-  --surface-primary: var(--ant-color-bg-container);
-  --surface-secondary: var(--ant-color-bg-layout);
-  --surface-tertiary: var(--ant-color-fill-alter);
-  --surface-overlay: var(--ant-color-bg-mask);
+  --surface-primary: var(--ant-color-bg-container, #ffffff);
```

Comment:
Adding default values effectively rolls back the use of `--ant-color-bg-container`, because now we still have to maintain one var definition for each theme, and it can get out of sync with AntD.

If there are some very specific use cases where variables defined on `.ant-layout` are not visible, then we can define colors just for those cases, without undermining the overall global approach for the majority of the components.

Reply:
See comments below, if you want a follow up PR to fix these things I can work on implementing it.